sentiment-api/service.bal

```diff
@@ -0,0 +1,56 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/openapi;
+
+@openapi:ServiceInfo {
+    title: "Sentiment Analysis Service",
+    version: "1.0.0",
+    description: "This service provides sentiment analysis for text data."
+}
+service /text\-processing on new http:Listener(9000) {
```

Comment:
```suggestion
service /analysis on new http:Listener(9000) {
```
Suggesting to keep the code look simple.

Reply:
[fdc5c9c](https://github.com/TharmiganK/ieee-hackathon-workshop/pull/1/commits/fdc5c9cb6fc7fb34e59140109996174f90abb924)